pubspec.yaml

```
@@ -107,14 +107,26 @@ flutter:
   uses-material-design: true
 
   assets:
+    - assets/Noto_Color_Emoji/LICENSE
     - assets/Source_Code_Pro/LICENSE.md
     - assets/Source_Sans_3/LICENSE.md
 
   fonts:
-    # Zulip's custom icons.  To use or edit, see class ZulipIcons.
-    - family: Zulip Icons
+    # Google's emoji font. (Web uses these emoji for the "Google" emojiset.)
+    #
+    # This should always be used under a condition that excludes iOS.
```

```suggestion
    # This should not be used on iOS.
```

I kept wanting to read this as being in the direction of saying it *should* be used, then realized the point was the situations where it shouldn't.

---

Also tweaked the next line:
```diff
-    # This should always be used under a condition that excludes iOS.
-    # iOS doesn't support this font in any of its available formats,
+    # This should not be used on iOS.
+    # iOS doesn't support any of the formats this font is available in,
```

because it sounded like "its available formats" might refer to iOS rather than the font.